datascience/src/pipeline/queries/monitorenv/control_units.sql

```diff
@@ -6,8 +6,9 @@ FROM control_units cu
 JOIN control_unit_contacts cuc
 ON cu.id = cuc.control_unit_id
 WHERE
-    cu.id IN :control_unit_ids
-    AND email IS NOT NULL
+    email IS NOT NULL
     AND is_email_subscription_contact
+    AND NOT archived
```

Comment:
thought: Je savais pas qu'on pouvait mettre du `NOT` après un booléen.

Reply:
yep !